modules/alloy/CHANGELOG.md

```diff
@@ -21,11 +21,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Changed disconnected components to log a warning instead of throwing an error when triggering or broadcasting events.
 - Renamed `LayoutInside` to `LayoutInset` to better reflect what it does.
 - Changed `LayoutInset` rendering behaviour, as it was inconsistent with other layouts. It will now mirror the `Layout` logic for each direction.
+- Renamed the `anchor` property in anchor specs to `type`.
```

Comment:
I renamed this to keep things consistent with our naming elsewhere. It's also super confusing when you do something like:
```
anchor: {
  anchor: 'node',
}
```